components/p3a/histograms_braveizer.cc

```diff
@@ -71,7 +71,7 @@ void HistogramsBraveizer::DoHistogramBravetization(
         answer = 1;
         break;
       default:
-        NOTREACHED_IN_MIGRATION();
+        NOTREACHED();
```

Comment:
it looks dangerous, upstream can potentially add new values to the historgram, let's not crash